lua/gatekeeper.lua

```diff
@@ -200,3 +229,37 @@ function init_iface(iface, name, ports, cidrs)
 	end
 	return ret
 end
+
+function init_lpm(gk_conf, net_conf, segments, gk_policies, gw_addrs)
+	local routes =
+		ffi.new("struct lua_ip_routes [" .. #segments .. "]")
+	for i, v in ipairs(segments) do
+		routes[i - 1].ip_addr = v["ip_addr"]
+		routes[i - 1].prefix_len = v["prefix_len"]
+		routes[i - 1].policy_id = v["policy_id"]
+	end
+
+	local policies =
+		ffi.new("struct lua_gk_policy [" .. #gk_policies .. "]")
+	for i, v in ipairs(gk_policies) do
+		policies[i - 1].policy_id = v["policy_id"]
+		policies[i - 1].action = v["action"]
+		policies[i - 1].is_gt_in_back_net = v["is_gt_in_back_net"]
+
+		if v["is_gt_in_back_net"] == false then
```

Comment:
Field `is_gt_in_back_net` should not be required in Lua since it can be computed. Pushing unnecessary work to the configuration is not a good idea.